apps/judicial-system/web/src/routes/CourtOfAppeal/AppealCase/AppealCase.tsx

```diff
@@ -88,138 +101,192 @@ const AppealCase = () => {
       assistant.value === workingCase.appealAssistant?.id,
   )
 
+  const sendNotifications = () => {
+    return sendNotification(
+      workingCase.id,
+      NotificationType.APPEAL_JUDGES_ASSIGNED,
+    )
+  }
+
   const previousUrl = `${constants.COURT_OF_APPEAL_OVERVIEW_ROUTE}/${id}`
-  const handleNavigationTo = (destination: string) =>
-    router.push(`${destination}/${workingCase.id}`)
+  const handleNavigationTo = async (destination: keyof stepValidationsType) => {
+    setNavigateTo(destination)
+
+    if (
+      hasSentNotification(
+        NotificationType.APPEAL_JUDGES_ASSIGNED,
+        workingCase.notifications,
+      )
+    ) {
```

Comment:
This means that no new notifications will be sent if the assistant or judges are changed. Is this the intended behaviour?

Reply:
Let's talk about this on refinement..